### PR TITLE
add support for EL9

### PR DESCRIPTION
**Pull request by jhoblitt**
As of 2022-11-21, there is no OBS repo for EL9. However, EL9 includes an lldpd package via appstream.

### CODE REVIEW

**Comment by jhoblitt** [comment]
Ubuntu 20.04 / puppet 6 is having some sort of x509 cert error which is repeatable but unrelated to the changes in this PR.

File: data/AlmaLinux-9.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (bastelfreak) [inline data/AlmaLinux-9.yaml:1]: IMO we should ads $osfamily-$version to hiera.yaml

Comment (jhoblitt) [reply data/AlmaLinux-9.yaml:1]: I actually prefer that too. Standby...

Comment (jhoblitt) [reply data/AlmaLinux-9.yaml:1]: Actually, that gets messy because of `path: '%{facts.os.name}-%{facts.os.release.major}.yaml'`. I will add a subdir.

Comment (jhoblitt) [reply data/AlmaLinux-9.yaml]: Done. I added `data/osfamily` and `data/osfamily/<major>` to try to prevent confusion.

**Review by ekohl — APPROVED** [review]

**Pull request merged**